### PR TITLE
RIA-3998 Fix nightly Jenkins build ia-aip-frontend

### PR DESCRIPTION
**Pull request by gavinslilley**
### JIRA link (if applicable) ###
[RIA-3998](https://tools.hmcts.net/jira/browse/RIA-3998)

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
     ])
 ])
 
 @Library("Infrastructure")
 
-def product = "ia"
+def type      = "nodejs"
+def product   = "ia"
 def component = "aip-frontend"
+def channel   = "#aip-tech"
```
Comment (makowalski) [inline Jenkinsfile_nightly:19]: let's use `ia-tech` channel, not sure if aip one is created

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -34,17 +41,17 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
...
-//  env.CCD_API_URL = 'http://ccd-data-store-api-aat.service.core-compute-aat.internal'
+    loadVaultSecrets(secrets)
+
+    env.TEST_URL = 'https://immigration-appeal.aat.platform.hmcts.net'
+    env.S2S_URL = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
+    env.CCD_API_URL = 'http://ccd-data-store-api-aat.service.core-compute-aat.internal'
+
+    enableCrossBrowserTest()
+    enableSecurityScan()    
+    env.S2S_MICROSERVICE_NAME = 'iac'
```
Comment (makowalski) [inline Jenkinsfile_nightly:55]: we can group together with envs

File: Jenkinsfile_nightly (hunk 1 above)
Comment (gavinslilley) [reply Jenkinsfile_nightly:19]: you are correct, this channel does not (or no longer) exist. have corrected and pushed

File: Jenkinsfile_nightly (hunk 2 above)
Comment (gavinslilley) [reply Jenkinsfile_nightly:55]: I made this change on the nightly-dev branch, also pushed this to guy now.

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**